### PR TITLE
Improved SSHResult

### PR DESCRIPTION
**Pull request by mfalesni**
namedtuple() was good, but I saw a way to simplify the testing code. Old behaviour is kept. New behaviour just gets out of your way.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit d6c83bce6196c3276af369857c05394c8fe2fbe5:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/ssh.py [hunk 1]
```diff
@@ -24,7 +24,46 @@
...
+
+class SSHResult(namedtuple("SSHResult", ["rc", "output"])):
+    """Allows rich comparison for more convenient testing.
+
+    Given you have ``result`` which is an instance of :py:class:`SSHResult`, you can do as follows:
+
+    .. code-block:: python
+
+        assert result  # If $?=0, then the result evaluates to a truthy value and passes the assert
+        assert result == 'installed'  # direct matching of the output value
```
Comment (psav) [inline utils/ssh.py]: Perhaps it would be good to have an "in" to check a substring also?
The Result is the output of the func right?

Comment (mfalesni) [inline utils/ssh.py]: That is also doable. Yes, result is an output of all operations that invoke commands.

**Comment by dajoRH** [comment]
Lint report for commit 35018fa306237ad7c25b73a0c02b711ec5ae280c:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/ssh.py (hunk 1 above)
Comment (mfalesni) [inline utils/ssh.py]: It is done.

**Pull request merged**